test/lib/redis-store-spec.js

```diff
@@ -463,3 +466,39 @@ describe('defaults', function () {
   });
 });
 
+describe('wrap function', function () {
```

Comment:
we should add some more tests for nonexistent keys (null, undefined) that we know are causing people issues #57